packages/react-day-picker/src/components/Dropdown/Dropdown.tsx

```diff
@@ -3,13 +3,19 @@ import React from 'react';
 import { IconDropdown } from 'components/IconDropdown';
 import { useDayPicker } from 'contexts/DayPicker';
 
+type DropdownOption = {
+  name: string;
+  value: string | number;
```

Comment:
Should be optional / undefined?

```suggestion
  value?: string | number | undefined;
```